src/ethereum/frontier/vm/memory.py

```diff
@@ -0,0 +1,92 @@
+"""
+Ethereum Virtual Machine (EVM) Memory
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. contents:: Table of Contents
+    :backlinks: none
+    :local:
+
+Introduction
+------------
+
+Implementation of the EVM memory expansion related operations.
+"""
+
+from typing import cast
+
+from ethereum.base_types import U256
+
+from . import Evm
+from .gas import GAS_MEMORY, subtract_gas
+
+
+def get_words(memory_size: int) -> int:
```

Comment:
I think `get` prefixes should generally refer to something that needs to be fetched  (cc: [snake charmers tactical manual](https://github.com/ethereum/snake-charmers-tactical-manual/blob/master/naming-things.md#prefixes)). 
A few alternatives (potentially alternate `compute` with `calc`): 
* `compute_aligned_index`
* `compute_max_words`
* `compute_padding`

Reply:
Strongly against `calc`.

Reply:
calculate?

Reply:
This won't be used anymore as the memory operations will be removed from this PR in favour of #250